### PR TITLE
Update manage-guest-access-in-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a not about adding a Guest user to an M365 group using Dynamic membership rules

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4202

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/admin/create-groups/manage-guest-access-in-groups.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ If you want to add a guest to the directory directly, you can [Add Azure Active
 
 If you want to edit any of a guest's information, you can [Add or update a user's profile information using Azure Active Directory](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-users-profile-azure-portal).
 
+> [!NOTE]
+> Dynamic membership is supported for Microsoft 365 Groups. If you want to add an external user to a M365 group using Dynamic membership rules, the option to "View all and manage members"  will not be available in the admin center. Instead you need to [create or update a dynamic group in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/enterprise-users/groups-create-rule)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/manage-guest-access-in-groups.md]:
```suggestion
> Dynamic membership is supported for Microsoft 365 Groups. If you want to add an external user to a Microsoft 365 group using dynamic membership rules, the option to "View all and manage members"  will not be available in the admin center. Instead, you need to [create or update a dynamic group in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/enterprise-users/groups-create-rule).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**